ipv-api/src/main/java/uk/gov/di/authentication/ipv/services/IPVAuthorisationService.java

```diff
@@ -1,13 +1,6 @@
 package uk.gov.di.authentication.ipv.services;
 
-import com.nimbusds.jose.EncryptionMethod;
-import com.nimbusds.jose.JOSEException;
-import com.nimbusds.jose.JWEAlgorithm;
-import com.nimbusds.jose.JWEHeader;
-import com.nimbusds.jose.JWEObject;
-import com.nimbusds.jose.JWSAlgorithm;
-import com.nimbusds.jose.JWSHeader;
-import com.nimbusds.jose.Payload;
```

Comment:
There was a section in the new developer guide about wildcards not being allowed:

> Wildcard imports are not allowed (eg. import com.nimbusds.openid.connect.sdk.* ). To prevent IntelliJ from automatically generating wildcard imports, in IntelliJ go to Settings > Editor > Code Style > Java. Set both Class count to use import with ‘*', and Names count to use static import with '*', to a large number (eg. 1000).

I'm not sure exactly why this is though

Reply:
Huh, I was just expecting spotless to sort that out for me...